qiskit/providers/basicaer/basicaertools.py

```diff
@@ -72,7 +72,7 @@ def cx_gate_matrix():
 
 
 def einsum_matmul_index(gate_indices, number_of_qubits):
-    """Return the index string for Numpy.eignsum matrix-matrix multiplication.
+    """Return the index string for Numpy.einsum matrix-matrix multiplication.
```

Comment:
This could probably be changed to `numpy.einsum` since the import is `numpy` not `Numpy` but not a big deal.